kube/dataservices/deployment.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: dataservices
+spec:
+    replicas: 2
+    selector:
+        matchLabels:
+            component: dataservices
+    template:
+        metadata:
+            labels:
+                component: dataservices
+        spec:
+            containers:
+                - name: dataservices
+                  image: your.container.registry.io/dataservices:latest
```

Comment:
I think it would be good in here to have some kind of identifier to  indicate that this is a bit of text that needs to be replaced. E.g:

- {{ your.container.registry.io }}
- ${ your.container.registry.io }
- <your.container.registry.io>
- [your.container.registry.io]
- etc

I don't have a strong opinion about which option we go with. Just something to make it clear that this is meant to be replaced in the final manifest.

Reply:
I guess I could be persuaded either way. I do think we need a way to tell a user that the files are examples and may need to be updated to fit their own configuration. I suppose that could even be done in the README

Reply:
I don't disagree that

`image: {{ YOUR_CONTAINER_REGISTRY }}/dataservices:latest`

doesn't stand out more, but it doesn't necessarily fit with the style of the rest of the document.  It has more of template style than example style.  I kind of stayed away from the template style because it is a little bit more work and I felt a bit more verbose when maybe it didn't have to be.  I did add a section in the README stating these are examples and should be used just as a reference.

Reply:
That's fair and your explanation makes sense re template vs example. Like I said, I don't have a particularly strong opinion on this and I did see the language in the README so I'm happy to proceed with this.